Dockerfile

```diff
@@ -1,3 +1,19 @@
+FROM alpine:latest AS build
```

Comment:
Can you specify at least the desired major version, just to avoid headaches from a floating tag later